source/connect.txt

```diff
@@ -22,11 +22,11 @@ Connect to MongoDB
    :titlesonly:
    :maxdepth: 1
     
-   /connect/mongoclient
-   /connect/connection-targets
-   /connect/connection-options
-   /connect/tls
-   /connect/stable-api
+   Create a MongoClient </connect/mongoclient>
+   Choose a Connection Target </connect/connection-targets>
+   Connection Options </connect/connection-options>
+   Enable TLS </connect/tls>
+   Stable API </connect/stable-api>
 
 ..    /connect/network-compression
 ..    /connect/server-selection
```

Comment:
Can we add labels to these as well?

Reply:
These are pages that haven't been created yet, so not necessary! @ltran-mdb2